.gitlab-ci.yml

```diff
@@ -54,6 +54,12 @@ workflow:
     - if [ -z "$DEPLOY_TAG" ]; then echo "DEPLOY_TAG must be set for an actionable build."; exit 1; fi
     # Latest tags need a confirmation var (provided manually: DEPLOY_LATEST)
     - if [[ "$DEPLOY_TAG" == "10.x-latest" ]] && [ -z "$DEPLOY_LATEST" ]; then echo "DEPLOY_LATEST must be set to progress with 'latest' tags."; exit 1; fi
+    # Install ahoy into the runner image.
+    - |
+      os=$(uname -s | tr [:upper:] [:lower:]) && architecture=$(case $(uname -m) in x86_64 | amd64) echo "amd64" ;; aarch64 | arm64 | armv8) echo "arm64" ;; *) echo "amd64" ;; esac) \
+        && wget -q https://github.com/ahoy-cli/ahoy/releases/latest/download/ahoy-bin-$os-$architecture -O /usr/local/bin/ahoy \
+        && chmod +x /usr/local/bin/ahoy
+    - apk add bash
```

Comment:
Bash is required for `ahoy` with default configuration; we can change this with `entrypoint` configuration however to keep this as similar as possible we'll install bash for now.